docs/Administration.rst

```diff
@@ -2433,6 +2433,121 @@ To check if your configuration takes effect, restart the server and
+
+-  :ref:`Manage spaces <AdministratingSpaces.ManageSpaces>`
+-  :ref:`Change permissions <AdministratingSpaces.Permissions>`
+
+.. _AdministratingSpaces.ManageSpaces:
+
+Manage spaces tab
+~~~~~~~~~~~~~~~~~~
+
+|image308|
+
+The "Manage spaces" tab is viewable by the group of users defined in the 
+:ref:`Permissions tab <AdministratingSpaces.Permissions>` by the "Manage spaces" permission. 
+Users with this permission are allowed to edit or delete any space in the platform.
+
+|image307|
+
+If the permission has no value defined in the interface, the group of users defined
+by the property :ref:`exo.social.spaces.administrators <SpacesAdministration>` in
+:ref:`exo.properties file <Configuration.eXoConfiguration>` is allowed to manage spaces.
+
+.. warning:: When the Manage spaces permission is defined both through the interface and 
+             by configuration, the value in the interface takes precedence. 
+
+.. note:: If the permission is neither defined through the interface nor by configuration, only administrators are
+          allowed to access the Manage spaces tab.
+          
+
+Edit space
+-----------
+
+Users with the "Manage Spaces" permission, can edit space settings by clicking the edit icon |image309| of a specific space.
+The Space Settings page of the target space is opened in another browser tab.
+
+|image320|
+
+
+From there, even without being a member on that space, one can edit the space settings, access, visibility, members, 
+applications and navigation. See :ref:`Managing space settings <Manage-Space-Settings>`
+section for more details.
+
+|image310|
+
+.. tip: You can find :ref:`here <Manage-Space-Settings>` the possible actions for a space's manager.
+
+Delete space
+--------------
+
+Users with the Manage Spaces permission, can also :ref:`delete <Delete-Space>` any space of 
+the platform even the spaces where they are neither a manager nor a member.
+
+To delete a space, a click on the delete icon |image311| of the corresponding space in the interface of 
+the "Manage spaces" tab is needed.
+
+|image312|
+
+A confirmation popup appears to confirm deletion of the space or cancel 
+:ref:`the space's deletion <Delete-Space>`.
+
+|image313|
+
+
+.. warning:: When you delete a space, all its data is also removed and can't be recovered.
+
+.. _AdministratingSpaces.Permissions:
+
+Permissions tab
+~~~~~~~~~~~~~~~~
+
+|image318|
+
+This tab allows administrators of the platform, define permissions related to all spaces of the platform.
+
+To define who can create spaces:
+
+1. Click on edit icon |image314| in the actions column of the line **Create spaces** ==> A field with the  
+   current values appears to easily edit them. The edit icon turns into a save icon |image315| and a new 
+   |image317| icon appears to cancel the changes.
+   
+   |image319|
+   
+2. Remove existing values and/or enter new ones. When typing in the field, type-ahead suggestions will appear below for easier selection
+   of the groups.
```

Comment:
@mzorai screenshot of the type-ahead suggestions would be useful here.

Reply:
@plamarque  i tried to make a screen shot for the type-ahead suggestions but when i click on the screen shot button they disappear!